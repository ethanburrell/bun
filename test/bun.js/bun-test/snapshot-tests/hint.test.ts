import {
    describe,
    expect,
    it,
  } from "bun:test";

describe("test hint", () => {
    it("hint should be well formatted", () => {
        expect("do not use a hint").toMatchSnapshot()
        expect("some string").toMatchSnapshot("using a hint")
    })
})
