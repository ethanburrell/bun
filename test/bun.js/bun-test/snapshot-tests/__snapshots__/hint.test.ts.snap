// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test hint hint should be well formatted: using a hint 1`] = `some string`;

exports[`test hint hint should be well formatted: using a hint 2`] = `another string`;

exports[`test hint hint should be well formatted 1`] = `do not use a hint`;
